### DISCUSSION TITLE
[Question] Loop over file extensions

### DISCUSSION THREAD

**Discussion by MaKyOtOx** [projectdiscovery/nuclei-templates#2261] [Q&A] [answered]
Hi Team,

I'm a rookie in writing templates and I'm looking for improving an existing template : 
Fhttps://github.com/projectdiscovery/nuclei-templates/blob/master/exposures/configs/docker-compose-config.yaml

By default, the docker-compose configuration filename supports both `.yml` and `.yaml` file extensions. What is the best way to loop over allowed extensions instead ? I didn't found the answer in the documentation examples (maybe I've missed it).
Thanks !!

**Comment by geeknik** [comment]
Simply change the path information in the template from this 
```
    path:
      - "{{BaseURL}}/docker-compose.yml"
      - "{{BaseURL}}/docker-compose.prod.yml"
      - "{{BaseURL}}/docker-compose.production.yml"
      - "{{BaseURL}}/docker-compose.staging.yml"
      - "{{BaseURL}}/docker-compose.dev.yml"
      - "{{BaseURL}}/docker-compose-dev.yml"
      - "{{BaseURL}}/docker-compose.override.yml"
```
to this
```
    path:
      - "{{BaseURL}}/docker-compose.yml"
      - "{{BaseURL}}/docker-compose.prod.yml"
      - "{{BaseURL}}/docker-compose.production.yml"
      - "{{BaseURL}}/docker-compose.staging.yml"
      - "{{BaseURL}}/docker-compose.dev.yml"
      - "{{BaseURL}}/docker-compose-dev.yml"
      - "{{BaseURL}}/docker-compose.override.yml"
      - "{{BaseURL}}/docker-compose.yaml"
      - "{{BaseURL}}/docker-compose.prod.yaml"
      - "{{BaseURL}}/docker-compose.production.yaml"
      - "{{BaseURL}}/docker-compose.staging.yaml"
      - "{{BaseURL}}/docker-compose.dev.yaml"
      - "{{BaseURL}}/docker-compose-dev.yaml"
      - "{{BaseURL}}/docker-compose.override.yaml"
```

**Comment by MaKyOtOx** [comment, marked as answer]
Yes obviously :/ I was looking for a smarter way to do it. By using Regex or substitution lists. Any thoughts ?

**Reply by ehsandeep** [reply]
@MaKyOtOx not possible in current version and will be supported once we implement this - https://github.com/projectdiscovery/nuclei/issues/533

**Reply by MaKyOtOx** [reply]
Thnks !
